packages/vocabulary/src/styles/button.scss

```
@@ -6,17 +6,22 @@ $button-family: $family-secondary;
   text-transform: uppercase;
   font-weight: 600;
   line-height: 0;
-  border: 0.125rem solid $color-dark-gray;
-  color: $color-dark-gray;
-  background-color: $color-white;
+  background-color: $color-tomato;
+  color: var(--button-color, #fff);
+  border: 0.125rem solid var(--button-border-color, transparent);
   font-size: 1.125rem;
   padding: $space-small $space-normal;
 
-  &:hover {
-    background-color: $color-dark-gray;
-    border-color: $color-dark-gray;
-    color: $color-white;
+  &:hover,
+  &:active,
+  &:focus {
+    color: #fff;
+    background-color: $color-brighter-tomato;
     text-decoration: none;
+    border-color: transparent;
+  }
+  &:focus, &:focus:not(:active) {
+    box-shadow: 0 0 0 0.125em var(--button-focus-outline, rgba(226,54,0,.25));
```

Small formatting fix, please just add spaces between the rbg values
```suggestion
    box-shadow: 0 0 0 0.125em var(--button-focus-outline, rgba(226, 54, 0, .25));
```